app/scripts/models/reliers/relier.js

```diff
@@ -91,7 +94,49 @@ define([
      */
     allowCachedCredentials: function () {
       return this.get('allowCachedCredentials');
+    },
+
+    /**
+     * Creates a resume token from subset of model fields.
+     */
+    getResumeToken: function () {
+      var resumeObj = {};
+      var fieldCount = 0;
+
+      _.each(this.getRelierFieldsInResumeToken(), function (itemName) {
+        if (this.has(itemName)) {
+          resumeObj[itemName] = this.get(itemName);
+          fieldCount++;
+        }
+      }, this);
+
+      if (fieldCount === 0) {
+        return null;
```

Comment:
not sure about this fieldCount thing. I think I'd rather have an empty object returned.